.github/scripts/publish-rhel.sh

```diff
@@ -1,59 +1,80 @@
 #!/bin/bash
 
-get_containers()
+# We used to give project id in the queries but the new API asks for _ID field of the project.
```

Comment:
I'm not sure it's a useful comment. Imagine you read this comment in a year, could you mind what was 2 years ago and if RH API has changed one year ago?

Reply:
I agree, I mainly put that comment for the reviewer and it has probably no use in a year or so. Do you think I should put links there so that people will see what has changed or should I delete that comment altogether?